src/inflators/media-loader.ts

```diff
@@ -28,4 +29,7 @@ export function inflateMediaLoader(
     MediaLoader.fileId[eid] = APP.getSid(fileId)!;
   }
   MediaLoader.src[eid] = APP.getSid(src)!;
+  if (linkSrc) {
+    MediaLoader.linkSrc[eid] = APP.getSid(linkSrc)!;
+  }
```

Comment:
`MediaLoader.linkSrc[eid]` needs to be initialized even if `linkSrc` is not set otherwise it can be problem if entity recycle happens that new entity can reuse linkSrc of the old one 

```typescript
// Assumes 0 indicates null string, not sure if this assumption is correct tho
MediaLoader.linkSrc[eid] = linkSrc !== undefined ? APP.getSid(linkSrc) : 0;
```